tools/generate-intrinsics-specs.sh

```diff
@@ -53,3 +53,11 @@ generate_rst_specs ./tools/intrinsic_db/mve.csv \
 
 # Check changes
 check_changes ./tmp/advsimd.new.rst ./neon_intrinsics/advsimd.rst
+check_changes ./tmp/mve.new.md ./mve_intrinsics/mve.md
+
+# Generate tmp file for pdfs
+generate_rst_specs ./tools/intrinsic_db/mve.csv \
+		   ./tools/intrinsic_db/mve_classification.csv \
+		   ./mve_intrinsics/mve.template.md \
+		   ./tmp/mve.for-pdf.md \
+		   pdf
```

Comment:
Shouldn't you follow this sequence:

1. generate the new specs
2. check the changes

Also, please note that in this script we should use the `markdown` workflow here, not the `pdf` one.

Reply:
Ah - I was wrong. You are already checking the newly generated markdown file. We just need to move the code that generates the PDF workflow in the scripts that creates the PDFs... sorry about that.